src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/AspireAIChat/.template.config/template.json

```diff
@@ -0,0 +1,349 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Common", "AI", "Web", "Aspire" ],
+  "identity": "Microsoft.Extensions.AI.Templates.AspireChat.CSharp",
+  "name": "AI + .NET Aspire Chat  App",
+  "description": "A project template for creating an AI chat application using .NET Aspire, which uses retrieval-augmented generation (RAG) to chat with your own data.",
+  "shortName": "aiaspirechat",
+  "defaultName": "AspireChatApp",
+  "preferNameDirectory": false,
+  // The placeholder sourceName needs to contain a dash to ensure the CSS bundle asset uses the assembly identity name
```

Comment:
Left this comment in for now until we know that we've addressed the issue raised here. I haven't tested it yet.